src/System Application/App/Guided Experience/app.json

```diff
@@ -88,6 +88,11 @@
       "id": "4af62c1c-2f6d-4574-ba34-bb5c149cdf93",
       "name": "Guided Experience Test",
       "publisher": "Microsoft"
+    },
+    {
+      "id": "d3433b68-4901-445f-9547-fdfeca57575a",
```

Comment:
I don't think we should encourage this, using internalsvisibleto can easily get out of hand.
Create an API in the GuidedExperience Facade and use ModuleInfo to block non-Microsoft from invoking it.

Reply:
True, I was just considering if this is something we would like to make internal or not. Probably no need and I will switch it to public.